randovania/games/am2r/exporter/joke_hints.py

```diff
@@ -2,21 +2,31 @@
 
 JOKE_HINTS = [
     "*This Wisdom Septogg does not seem very wise.*",
+    "*How many Ridley's have you found so far in this game?*",
     "*Movement in Morph Ball is faster than unmorphed, except for Speedboosting.*",
+    "*Samus Aran... You must exterminate the Metroids once and for all!*"
     "*Posture check! Stretch your wrists, flex your fingers, and straighten your back.*",
+    "*Ma'am, thou'rt mistaken, I'm not a pet, I'm a Septogg and master bug eater.*",
     "*Why so Serris?*",
     "*Adam has not yet authorized the use of this hint.*",
     "*Did you know that Frog Legs are delicious with some Goat Butter, Garlic and Parsley?*",
     "*If you see an Omega Metroid shinesparking towards your general direction, run.*",
     "*For hints to Return... we must get rid of Proteus Ridley, find the Metroid DNA, and repair the Chozo Seals...*",
+    "*Have you ever thought about Zeta Metroid Jockeys? Well, now you have.*",
     "*Judging by the architecture on SR-388, the Chozo's favourite transportation device seem to be TUUUUUUUUUBES!*",
+    "*Only a FOOL would expect me to give you useful hints.*",
+    "*Space Jump, Ice, Bombs? You want it? It's yours, my friend, as long as you have enough credits.*",
     "*S P E E N*",
+    "*Wisdom... is everything.*",
     "*You are faced with an overwhelming amount of cute Septogg noises. Accept your helplessness.*",
+    "*Sorry, I do not have much to say. Thanks for talking to me though.*",
     "*Hear the words of O-Lir, last Sentin- wait, wrong game.*",
     "*Skippy is on an epic quest to unravel its true destiny.*",
     "*Uh oh! It seems your Metroid can't crawl, please contact Wisdom Septogg S.A. for help.*",
+    "*Maybe the Metroid DNA were the friends we made along the way...*",
```

Comment:
I think this should be was.

Reply:
its supposed to be plural, were sounds nicer for my brain here.

Reply:
eh, using was just in case